### PR TITLE
Doc revisions to OPA page

### PR DESCRIPTION
**Pull request by artberger**
See issue https://github.com/solo-io/gloo/issues/5640

* Removes unmaintained script
* General doc revisions for style and grammar

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- N/A If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [x] I opened a draft PR or added the work in progress label if my PR is not ready for review
- N/A I have performed a self-review of my own code
- N/A I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- N/A I have added tests that prove my fix is effective or that my feature works

BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/5640

### CODE REVIEW

File: docs/content/guides/security/opa/_index.md [hunk 1]
```diff
@@ -4,86 +4,94 @@ weight: 80
...
 
-Before you get started, you will need to have Gloo Edge running in a Kubernetes cluster. For more information, you can follow this [installation guide]({{% versioned_link_path fromRoot="/installation/gateway/kubernetes/" %}}).
+---
 
-### Setup
+## Before you begin
 
-First, setup OPA as a validating web hook. In this mode, OPA validates the Kubernetes objects before they are visible to the controllers that act on them (Gloo Edge in our case).
+1. [Install Gloo Edge in a Kubernetes cluster]({{% versioned_link_path fromRoot="/installation/gateway/kubernetes/" %}}).
+2. [Follow the OPA documentation](https://www.openpolicyagent.org/docs/latest/kubernetes-tutorial/) to set up OPA as an admission controller in your cluster. In this mode, OPA validates Kubernetes objects before they become visible to other controllers that act on them, including Gloo Edge.
```
Comment (Rachael-Graham) [inline docs/content/guides/security/opa/_index.md:23]: in what mode? is that something covered in that doc

File: docs/content/guides/security/opa/_index.md [hunk 2]
```diff
@@ -4,86 +4,94 @@ weight: 80
...
-(3) This object has a prefixRewrite stanza.
-
-If all these conditions are true, the object will be denied with this message:
-```
-	msg := "prefix re-write not allowed"
-```
+| Part of the policy | Description |
+| ------------------ | ----------- |
+| operations = {"CREATE", "UPDATE"} | Applies the policy only to objects that are created or updated. |
+| deny[msg] { | Starts a policy to deny to creating or updating objects, if all of the conditions in the braces are met. |
```
Comment (Rachael-Graham) [inline docs/content/guides/security/opa/_index.md:77]: does this need a closing brace or nah

File: docs/content/guides/security/opa/_index.md [hunk 3]
```diff
@@ -161,10 +169,10 @@ For your convenience, here's the content of teardown.sh:
 
 ## Next Steps
 
-Now that you've see how to configure a basic policy with OPA, you can go further down the rabbit hole of policies, or check out some of the other security features in Gloo Edge.
+Now that you know how to configure a basic policy with OPA, you can go further down the rabbit hole of policies, or check out some of the other security features in Gloo Edge.
```
Comment (Rachael-Graham) [inline docs/content/guides/security/opa/_index.md:172]: maybe "you can continue learning about policies"?

File: docs/content/guides/security/opa/_index.md (hunk 1 above)
Comment (artberger) [reply docs/content/guides/security/opa/_index.md:23]: Yeah, as an admission controller.

**Review by Rachael-Graham — APPROVED** [review]

**Pull request merged**